cff.el

```diff
@@ -59,12 +59,18 @@
 (eval-when-compile (require 'cl-lib))
 ;; optional helm dependency
 (require 'helm nil t)
+(require 'ivy nil t)
 
 (defvar cff-use-helm-choice t
   "Determines what to do if there are several possible choices.
 If t, use helm if available.
 If nil, always use internal choice mode.")
 
+(defvar cff-use-ivy-choice t
```

Comment:
you have set both of these variables to `t`. Since it is a new feature, I would suggest to turn it off by default to not to break backwards compatibility. You would turn it on via setq in your init file.
After the change is done and taken, I'll remove this variable (and notify you) and introduce some generic variable which could be set to either helm, ivy or internal, while keeping the original cff-use-helm-choice only for backwards compatibility.